app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -133,17 +157,32 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private fun buildDrawer(context: Activity, toolbar: Toolbar, accountHeader: AccountHeader,
                             savedInstanceState: Bundle?): Drawer {
-        return DrawerBuilder(context)
+        val stickyDrawerItems = generateStickyDrawerItems()
+        this.stickyItemIds = stickyDrawerItems.mapNotNull { DrawerItem.fromOrNull(it.identifier) }
+                .toTypedArray()
```

Comment:
I didn't inline `generateStickyDrawerItems` into the call below because I still need the return value in the drawer builder below and I don't want to call the method twice.